electron/ironfish/AccountManager.ts

```diff
@@ -282,10 +284,21 @@ class AccountManager
     }
   }
 
-  async submitAccount(createParams: AccountValue): Promise<WalletAccount> {
-    const newAccount = await this.node.wallet.importAccount(createParams)
-
-    return newAccount.serialize()
+  async submitAccount(createParams: AccountValue): Promise<ImportResponse> {
+    const client = await this.sdk.connectRpc()
```

Comment:
I would propose to connect to RPC after node initialization in ironfish manager and store it in the class property, so then it can be passed to other managers instead of  sdk.

Reply:
This actually turned out to be a pain because the node initialization/rpc client initialization has to be in a very particular fragile order. When we refactor app to get rid of internal node, we can propagate RPC everywhere.